runtime/legion/legion_views.cc

```diff
@@ -4327,7 +4509,8 @@ namespace Legion {
                               VersionTracker *src_version_tracker,
                   const LegionMap<ApEvent,FieldMask>::aligned &preconditions,
                         LegionMap<ApEvent,FieldMask>::aligned &postconditions,
-                        PredEvent guard, CopyAcrossHelper *across_helper) const
+                        PredEvent guard, PhysicalTraceInfo &trace_info,
```

Comment:
Yes, this is what I was expecting to see everywhere that we have a 'bool tracing' in other method calls.